agent/engine/docker_task_engine_test.go

```diff
@@ -25,23 +25,32 @@ import (
 	"github.com/aws/amazon-ecs-agent/agent/credentials/mocks"
 	"github.com/aws/amazon-ecs-agent/agent/engine/dockerclient"
 	"github.com/aws/amazon-ecs-agent/agent/engine/testdata"
+	"github.com/aws/amazon-ecs-agent/agent/eventstream"
 	"github.com/aws/amazon-ecs-agent/agent/statemanager/mocks"
 	"github.com/aws/amazon-ecs-agent/agent/utils/ttime/mocks"
 	"github.com/aws/aws-sdk-go/aws"
 	docker "github.com/fsouza/go-dockerclient"
 	"github.com/golang/mock/gomock"
+
+	"golang.org/x/net/context"
 )
 
 const credentialsId = "credsid"
 
 var defaultConfig = config.DefaultConfig()
+var containerChangeEventStream *eventstream.EventStream
+
+func init() {
```

Comment:
You have a single event stream that goes globally in all tests.  Can you move this to a per-test scope?  Or alternately just use a mock, since it doesn't look like this is actually important for these tests?